db/config.hh

```diff
@@ -111,6 +111,7 @@ struct experimental_features_t {
         BROADCAST_TABLES,
         KEYSPACE_STORAGE_OPTIONS,
         TABLETS,
+        AUTH_V2,
```

Comment:
So I am confused by now. Didn't we decide to not add experimental flag for this feature and include migration code instead?

Reply:
Yes we'll need to use the consistent-topology-changes feature instead of auth-v2 feature (and remove auth-v2 feature)

Reply:
Yes, as I understand it was Avi's condition for merge

Reply:
I see reference here slipped through, removed